R/zchunk_batch_all_aglu_emissions_IRR.xml.R

```diff
@@ -1,14 +1,14 @@
-#' module_emissions_batch_all_aglu_emissions_IRR.xml
```

Comment:
Wait...why are there two xml-generating files here? You should just need `zchunk_batch_all_aglu_emissions_IRR_MGMT.xml.R` I think?